userspace/falco/stats_writer.cpp

```diff
@@ -357,6 +357,43 @@ void stats_writer::collector::get_metrics_output_fields_wrapper(
 		metric_name_file_sha256 = "falco.sha256_config_file." + falco::utils::sanitize_metric_name(metric_name_file_sha256);
 		output_fields[metric_name_file_sha256] = item.second;
 	}
+
+	if (stats_snapshot_time_delta_sec == 0)
```

Comment:
This new trick helps to create the json only once every inspector reload. For Prometheus however it does it every time right now @sgaist.